commcare/src/main/resources/webapp/js/filters.js

```diff
@@ -0,0 +1,19 @@
+(function () {
+    'use strict';
+
+    /* Filters */
+
+    var filters = angular.module('commcare.filters', []);
```

Comment:
I guess this is fine, maybe even better than sending unknown json fields through http